docs/GameEngines.rst

```diff
@@ -0,0 +1,134 @@
+    :height: 304px
+
+Open your Unity project and copy **Managed(C#)** libraries from BrainFlow and **all dependencies** to the Assets folder, after that copy **Unamanaged(C++)** libraries from BrainFlow to the root folder of your project.
+
+.. image:: https://live.staticflickr.com/65535/50095628822_14538fede0_c.jpg
+    :width: 800px
+    :height: 402px
+
+
+Now, you are able to use BrainFlow API in your C# scripts!
+
+For demo we will create a simple script to read data and calculate band powers.
+
+.. code-block:: csharp 
+
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using UnityEngine;
+
+    using Accord;
+    using Accord.Math;
+    using brainflow;
+
+    public class Sphere : MonoBehaviour
+    {
+        private BoardShim board_shim = null;
+        private int sampling_rate = 0;
+        private int[] eeg_channels = null;
+
+        // Start is called before the first frame update
+        void Start()
+        {
+            try
+            {
+                BoardShim.set_log_file("brainflow_log.txt");
+                BoardShim.enable_dev_board_logger();
+
+                BrainFlowInputParams input_params = new BrainFlowInputParams();
+                int board_id = (int)BoardIds.SYNTHETIC_BOARD;
+                board_shim = new BoardShim(board_id, input_params);
+                board_shim.prepare_session();
+                board_shim.start_stream(450000, "file://brainflow_data.csv:w");
+                
+                sampling_rate = BoardShim.get_sampling_rate(board_id);
+                eeg_channels = BoardShim.get_eeg_channels(board_id);
+                Debug.Log("Brainflow streaming was started");
+            }
+            catch (BrainFlowException e)
+            {
+                Debug.Log(e);
+            }
+        }
+
+        // Update is called once per frame
+        void Update()
+        {
+            if (board_shim == null)
+            {
+                return;
+            }
+            int number_of_data_points = DataFilter.get_nearest_power_of_two(sampling_rate);
+            double[,] data = board_shim.get_current_board_data(number_of_data_points);
+            if (data.GetRow(0).Length < number_of_data_points)
+            {
+                // wait for more data
+                return;
+            }
+            // calc bandpowers per channel
+            for (int i = 0; i < eeg_channels.Length; i++)
+            {
+                Tuple<double[], double[]> psd = DataFilter.get_psd(data.GetRow(eeg_channels[i]), 0,
+                    data.GetRow(eeg_channels[i]).Length, sampling_rate, (int)WindowFunctions.HANNING);
+                double band_power_alpha = DataFilter.get_band_power(psd, 7.0, 13.0);
+                double band_power_beta = DataFilter.get_band_power(psd, 14.0, 30.0);
+                Debug.Log("Alpha/Beta Ratio:" + (band_power_alpha / band_power_beta));
+            }
+        }
+
+        private void OnApplicationQuit()
```

Comment:
I think OnDestroy() is more correct if you want to be perfectly symmetrical with Start() for resource management.